### PR TITLE
[Examples and Tests] Tick Single Cluster

### PR DESCRIPTION
**Pull request by tonerdo**
This PR adds an example for demonstrating how to deploy all TICK components* on a single AMI. It also adds accompanying automated tests.

(*) Kapacitor will be included at a later date

### CODE REVIEW

File: examples/tick-single-cluster/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+module](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/modules/load-balancer).
+
+You will need to create an [Amazon Machine Image (AMI)](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) 
+that has all components of the TICK stack installed, which you can do using the [tick-ami 
+example](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples/tick-ami)). 
+
+To see an example of TICK deployed across separate clusters, see the [tick-multi-cluster
+example](https://github.com/gruntwork-io/terraform-aws-influx/blob/master/examples/tick-multi-cluster).
+
+## What resources are does this example deploy?
```
Comment (robmorgan) [inline examples/tick-single-cluster/README.md]:
```suggestion
## What resources does this example deploy?
```

File: examples/tick-single-cluster/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+1. A single _all in one server_ behind an ASG where we run 
+    [telegraf](/modules/run-telegraf), [influxdb](/modules/run-influxdb),
+    [chronograf](/modules/run-chronograf) and [kapacitor](/modules/run-kapacitor)
+1. An [Application Load Balancer](https://github.com/gruntwork-io/module-load-balancer)
+
+You will need to create [Amazon Machine Images (AMIs)](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) 
+that have all of the ELK components installed. You can do this using:
```
Comment (robmorgan) [inline examples/tick-single-cluster/README.md]:
```suggestion
that have all of the TICK-stack components installed. You can do this using: 
```
?

File: examples/tick-single-cluster/main.tf [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+  # source = "git::git@github.com:gruntwork-io/terraform-aws-influx.git//modules/influxdb-cluster?ref=v0.0.1"
+  source = "../../modules/influxdb-cluster"
+
+  cluster_name = "${var.cluster_name}"
+  min_size     = 1
+  max_size     = 1
+
+  # We use small instance types to keep these examples cheap to run. In a production setting, you'll probably want
+  # R4 or M4 instances.
+  instance_type = "t2.micro"
```
Comment (brikis98) [inline examples/tick-single-cluster/main.tf]: The whole TICK stack successfully runs on a `t2.micro`? If so, that's awesome.

File: examples/tick-single-cluster/main.tf [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+  # When using these modules in your own code, you will need to use a Git URL with a ref attribute that pins you
+  # to a specific version of the modules, such as the following example:
+  # source = "git::git@github.com:gruntwork-io/terraform-aws-influx.git//modules/load-balancer-target-group?ref=v0.0.1"
+  source = "../../modules/load-balancer-target-group"
+
+  target_group_name    = "${var.cluster_name}-itg"
+  asg_name             = "${module.tick.asg_name}"
+  port                 = "${module.influxdb_security_group_rules.api_port}"
+  health_check_path    = "/ping"
+  health_check_matcher = "204"
```
Comment (brikis98) [inline examples/tick-single-cluster/main.tf]: Why 204? Might want to add link to relevant docs.

Comment (tonerdo) [reply examples/tick-single-cluster/main.tf]: InfluxDB specifically has a ping endpoint for checking its health. I'll link to it

**Pull request merged**